frontend/src/components/XML/index.tsx

```diff
@@ -0,0 +1,18 @@
+import { AmountInfoCard } from "components/AmountInfoCard";
+import { SectionCard } from "components/SectionCard";
+
+import "./styles.scss";
+
+export const XML = () => {
+  return (
+    <SectionCard title="XML">
+      <div className="XMLContainer">
+        <AmountInfoCard title="Average Transaction Fee" amount="0.0002 XML" />
```

Comment:
`0.0002 XML` shouldn't be hardcoded. Let's get the average over the last 30 days, from the backend `/api/fees/stats` endpoint

Reply:
no let's hardcode those two, according to the [ticket](https://github.com/stellar/dashboard/issues/273)